x-pack/heartbeat/monitors/browser/browser.go

```diff
@@ -27,6 +28,8 @@ func create(name string, cfg *config.C) (p plugin.Plugin, err error) {
 	// We don't want users running synthetics in environments that don't have the required GUI libraries etc, so we check
 	// this flag. When we're ready to support the many possible configurations of systems outside the docker environment
 	// we can remove this check.
+	ev := os.Environ()
+	logp.L().Info("EV %v", ev)
```

Comment:
Do we want to keep this log line here? It feels to me we shouldn't log all the env. vars to `stdout` as that could leak information to logs.

Reply:
No, good catch! I thought I'd pulled all of those